src/sections/FAQ/index.tsx

```diff
@@ -1,22 +1,18 @@
-import Faq from '../../components/Faq/index'
-import faqData from './faq.jsx'
-import s from './style.module.scss'
-
+import Faq from '../../components/Faq/index';
+import faqData from './faq.jsx';
+import s from './style.module.scss';
 
 const FAQ: React.FC = () => {
-
   return (
-    <div className = {s.background}>
-      <div className = {s.faqContainer}>
-        <h2 className = {s.faqTitle}>FAQs</h2> 
-        <div className = {s.faqBox}>
-          <Faq
-            data={faqData.students}
-          />
+    <div className={s.background} id={'faq'}>
```

Comment:
The other sections used `id="companies"` but here it uses `id={'faq'}` which is somewhat inconsistent albeit fine